src/client/pages/Kiosk/Kiosk.scss

```diff
@@ -1,5 +1,3 @@
-@import 'src/client/style/partials';
```

Comment:
Removing these imports as the Kiosk doesn't really follow our established conventions fully. Also, in the future we could ideally have the Kiosk bundle be fetched on its own, and for that better have it as independent as possible from the platform. Maybe that can be accomplish when we migrate to Vite and we can avoid fetching the main bundle unnecessarily.